src/App.tsx

```diff
@@ -89,11 +89,14 @@ export const App: React.FC = () => {
             />
           )}
         />
-        {/* <Route
+        <Route
           exact
           path='/markets/:id'
-          render={({ match }) => <Details id={parseInt(match.params.id)} />}
-        /> */}
+          render={({ match }) => {
+            const { id } = match.params
```

Comment:
Interesting. I've only previously seen this sent in as props this way: `id={match.params.id}`, but if it works it works! Thank you benevolent JavaScript overlords.

Reply:
yep - went off how Em and I completed the card clicks in RT